SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -12,6 +12,8 @@ public class I18nManager {
 
     public static var instance = I18nManager()
     
+    public static var fallbackLanguage: String?
+    
     private init() {}
     
     public var defaultLanguage: String?
```

Comment:
```suggestion
    public static var instance = I18nManager()
    private init() {}
    
    public var defaultLanguage: String?
    public var fallbackLanguage: String?
```

This does not need to be static anymore. Let's remove that so that the call site stays same for the fallback as it is for the default.